tools/validate_paths.go

```diff
@@ -230,14 +248,41 @@ func checkFiles(knownOC map[string]pathType, files []string) ([]file, error) {
 			}
 		}
 
-		if len(lines) == 0 && len(errs) == 0 {
+		if len(lines) == 0 && len(errs) == 0 && len(dependencies) == 0 {
 			continue
 		}
-		report = append(report, file{name: f, lines: lines, errors: errs})
+		report = append(report, file{name: f, lines: lines, dependencies: dependencies, errors: errs})
 	}
+	report = validateDependency(validProfile, report)
 	return report, nil
 }
 
+// getFeatureProfileNameFromPath gets feature profile id.name from path
+func getFeatureProfileNameFromPath(file string, fp fppb.FeatureProfile) string {
```

Comment:
protobuffers should be passed by pointer (they contain a mutex, which shouldn't be copied)